lighthouse-cli/test/fixtures/dobetterweb/third_party/aggressive-promise-polyfill.js

```diff
@@ -9,5 +9,3481 @@
+// vendored from zone.js@0.11.4
+// purposefully not minified to trigger unminified-javascript
```

Comment:
is this still an _aggressive_ promise polyfill? It might be better to bump back down to 0.7.3 unless you verified, since that's a lot of (potentially) breaking changes since #1178